pkg/controller/disruption/disruption.go

```diff
@@ -705,11 +721,8 @@ func (dc *DisruptionController) trySync(ctx context.Context, pdb *policy.PodDisr
 	}
 	// We have unmamanged pods, instead of erroring and hotlooping in disruption controller, log and continue.
 	if len(unmanagedPods) > 0 {
-		klog.V(4).Infof("found unmanaged pods associated with this PDB: %v", unmanagedPods)
-		dc.recorder.Eventf(pdb, v1.EventTypeWarning, "UnmanagedPods", "Pods selected by this PodDisruptionBudget (selector: %v) were found "+
-			"to be unmanaged or managed by an unsupported controller (missing a /scale subresource). As a result, the status of the PDB "+
-			"cannot be calculated correctly, which may result in undefined behavior. To account for these pods please set \".spec.minAvailable\" "+
-			"field of the PDB to an integer value.", pdb.Spec.Selector)
+		logger.V(4).Info("Found unmanaged pods associated with this PDB:",
+			strings.Join(unmanagedPods, ",'"))
```

Comment:
Try running "hack/verify-golangci-lint.sh", it should complain about this (assuming that the base of your branch is recent enough).

```suggestion
		logger.V(4).Info("Found unmanaged pods associated with this PDB", "pods", unmanagedPods)
```